gax/src/test/java/com/google/api/gax/batching/BatcherImplTest.java

```diff
@@ -614,6 +671,73 @@ public boolean isLoggable(LogRecord record) {
     }
   }
 
+  /**
+   * Validates the absence of warning in case {@link BatcherImpl} is garbage collected after being
+   * closed.
+   *
+   * <p>Note:This test cannot run concurrently with other tests that use Batchers.
```

Comment:
Is there some code somewhere that prevents this from running concurrently? usually that depends on the test runner.

Reply:
no, but I have some ambitions to parallelizing the tests in the future, so wanted call out that this test is not self contained